gm4_trash_cans/data/gm4_trash_cans/functions/disable/wet.mcfunction

```diff
@@ -0,0 +1,18 @@
+# @s = trash_can
+# run from check_dropper
+
+# Water cools down the magma block
+execute if block ~ ~-1 ~ water run tag @s add gm4_trash_disable
+execute if block ~ ~1 ~ water run tag @s add gm4_trash_disable
+execute if block ~-1 ~ ~ water run tag @s add gm4_trash_disable
+execute if block ~1 ~ ~ water run tag @s add gm4_trash_disable
+execute if block ~ ~ ~-1 water run tag @s add gm4_trash_disable
+execute if block ~ ~ ~1 water run tag @s add gm4_trash_disable
+
+# Visuals
+execute if entity @s[tag=gm4_trash_disable] run particle dolphin ~ ~1.2 ~ .05 0 .05 0.1 3
+execute if entity @s[tag=gm4_trash_disable] run data merge entity @s {CustomName:'"gm4_trash_can"',ArmorItems:[{},{},{},{id:"minecraft:cobblestone",Count:1b,tag:{CustomModelData:1}}]}
+
+# Change Tag
+tag @s[tag=gm4_trash_disable] add gm4_trash_disabled
+tag @s[tag=gm4_trash_disable] remove gm4_trash_disable
```

Comment:
Why do you use these two oddly similar tags in the first place?
Assuming this is so that the "disabling" particle is only played once, this probably won't work as tags aren't exclusive.

Reply:
- The purpose is so you don't have to worry about lava burning down your house. It can sit in the middle of your wooden house without the risk of destroying anything.
- Quite a few people have said that it's annoying walking around spawn and randomly picking up tons of garbage items that somebody dropped. Letting them despawn is messy and annoying.
- You say one block solution as if lava isn't already just one block. (again lava burns things like your house)

Reply:
There are 2 different particles that can play when it is disabled. One for netherite items being inside and another for it being "wet". They get the tag "disable" then it plays the particle and finally converts it to "disabled" so the other particle doesn't get played.